typespec-extension/src/code-model-builder.ts

```diff
@@ -511,6 +512,12 @@ export class CodeModelBuilder {
     // check for generating protocol api or not
     codeModelOperation.generateProtocolApi = shouldGenerateProtocol(this.sdkContext, operation);
 
+    // currently we treat @Internal the same as @convenientAPI(false) and @protocolAPI(false) combined
+    if(isInternal(this.sdkContext, operation)) {
+      codeModelOperation.generateProtocolApi = false;
+      codeModelOperation.convenienceApi = undefined;
+    }
```

Comment:
move it before https://github.com/Azure/autorest.java/blob/e53b5f2f1773b0c5dbefa971f62734589e32f986/typespec-extension/src/code-model-builder.ts#L453

other procedures already checking `convenienceApi` for schema tracking. E.g. 
https://github.com/Azure/autorest.java/blob/e53b5f2f1773b0c5dbefa971f62734589e32f986/typespec-extension/src/code-model-builder.ts#L720-L722

Reply:
It means the model should not be generated, if only used in `@internal` operation.